common/service/dynamicconfig/constants.go

```diff
@@ -236,6 +239,8 @@ const (
 	DomainName
 	// TaskListName is the tasklist name
 	TaskListName
+	// TaskType is the task type (0:Decision, 1:Activity)
+	TaskType
```

Comment:
Probably you need to have a filters item for TaskType.

Reply:
Good catch, addressed. Will also change and strengthen unit test for these later